### ISSUE TITLE
coveralls.io failures cause build failures

### ISSUE THREAD

**Issue by dpkp** [dpkp/kafka-python#353]
Test runs fail and end w/ this in logs:

```
Submitting coverage to coveralls.io...

```

we need to fix this so coveralls service failures dont affect our build pass/fail results

**Comment by alisaifee** [comment]
sorry about the noise here, I typed the wrong issue number.

**Comment by zackdever** [comment]
From https://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build:

> The `after_success`, `after_failure`, `after_script` and subsequent stages do not affect the the build result.

    $ tail -n 2 .travis.yml
    after_success:
      - coveralls

Looks like this should never happen. I looked through the Travis build history and couldn't find any builds that failed with `Submitting coverage to coveralls.io...`. If we had a link to an old build that showed the build failure from an `after_success` command, then we should file a ticket with Travis CI. 

I did find a few builds that failed with `The command "pip install tox coveralls" failed and exited with 2 during .` e.g. https://travis-ci.org/dpkp/kafka-python/jobs/49398963. That didn't fail because of coveralls, but it is a possibility. To be cautious we could move the coveralls pip install to the `after_success` section.
